### DISCUSSION TITLE
description of films or series in a language other than English

### DISCUSSION THREAD

**Discussion by Dolphinyara** [seerr-team/seerr#567] [Ideas]
Since I’m setting up the resource in many ways, it would be more convenient for parents if the descriptions of films and TV series could be chosen in language, is it possible to implement this? (For example, Ukrainian or Russian)

**Comment by fallenbagel** [comment]
This is already possible. Settings > display language.

![Screenshot_20231117_145321_Chrome Dev.jpg](https://github.com/Fallenbagel/jellyseerr/assets/98979876/2395ea5e-cff0-499f-86d1-220fb1f34842)

![Screenshot_20231117_145329_Chrome Dev.jpg](https://github.com/Fallenbagel/jellyseerr/assets/98979876/13ce0b44-c3a5-4d9a-a58e-bf71714fa2b5)

**Reply by Dolphinyara** [reply]
Thank you, the Russian language really works.

**Reply by fallenbagel** [reply]
> Thank you, the Russian language really works.

Ukrainian is also added now and will be in the next release. #525
